watsononlinestore/watson_online_store.py

```diff
@@ -593,7 +692,7 @@ def handle_list_shopping_cart(self):
         shopping_list = self.cloudant_online_store.list_shopping_cart(cust)
         for index, item in enumerate(shopping_list):
             formatted_out += str(index+1) + ") " + \
-                             str(item.encode('utf-8')) + "\n"
+                str(item.encode('utf-8')) + "\n"
```

Comment:
I think this blows up with non-ascii chars in py2. Why are we doing this encode and then concatenating with strings? encode does allow error replace or ignore. Maybe we need that?